cmd/juju/application/removesaas.go

```diff
@@ -71,11 +77,18 @@ func (c *removeSaasCommand) Init(args []string) error {
 	return nil
 }
 
+func (c *removeSaasCommand) SetFlags(f *gnuflag.FlagSet) {
+	c.ModelCommandBase.SetFlags(f)
+	f.BoolVar(&c.Force, "force", false, "Completely remove an application and all its dependencies")
```

Comment:
Are we talking about 'applications' in remove-saas?.. I wonder if it makes sense here or if there s another term that is more natural?

Reply:
I think that the term "applications" is most natural. It's included in `juju help remove-saas`:

> Summary:
> Remove consumed applications (SAAS) from the model.

> Details:
> Removing a consumed (SAAS) application will terminate any relations that
> application has, potentially leaving any related local applications
> in a non-functional state.